rules/linux/lnx_install_root_certificate.yml

```diff
@@ -0,0 +1,22 @@
+title: Install Root Certificate
+id: 78a80655-a51e-4669-bc6b-e9d206a462ee
+description: Detects installed new certificate
+references:
+    - https://attack.mitre.org/techniques/T1553/004/
+author: Ömer Günal, oscd.community
+date: 2020/10/05
+tags:
+    - attack.defense_evasion
+    - attack.t1553.004
+level: low
+logsource:
+    product: linux
+detection:
+    keywords:
+        - keys|contains|all:
+          - 'mv '
+          - '/usr/local/share/ca-certificates'
+          - 'update-ca-certificates'
```

Comment:
there are a few problems with `keys|contains|all`. let me go through them one by one:

1. it is not going to work as the Linux audit log, and all the other logging tools (i.e. osquery, auditbeat etc) are monitoring *syscalls*, which means that execution of the test:

```
mv #{cert_filename} /usr/local/share/ca-certificates
echo sudo update-ca-certificates
```
is going to produce two events. and since you put all command lines under `|contains|all`, it's not going to work, as this logic is looking for all strings in one event. It will work if you will split them into two `selections`

2. The `key` is not going to work as there is no such field in process execution events (audit log for `execve` syscall, osquery `proces_events` or `processes` tables, etc). I would rather use the following construction:

```
detection:
    selection:
        - CommandLine|contains|all:
          - 'mv '
          - '/usr/local/share/ca-certificates'
```

3. This rule is not going to cover [Atomic Test #1 - Install root CA on CentOS/RHEL](https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1553.004/T1553.004.md#atomic-test-1---install-root-ca-on-centosrhel), as it uses different commands for CentOS/RHEL. Could you please update you rule so it would cover both atomic tests for Debian/Ubuntu and CentOS/RHEL?

Thank you!

Reply:
Hello @omergunal! Could you please fix this?

Reply:
Could you also please finish this part?

Reply:
Thanks!